otwcode/otwarchive
# AO3-4313 Load work's first posted chapter for everyone but admins and co-creators

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-4313

## Purpose

Invited/approved co-creators and admins can now view draft works

## Testing Instructions

1. create a multi-chapter draft work and add a co-creator
2. Log in as co-creator and go to co-creator requests page
3. Open draft work you were invited to (without accepting the invite)
4. You should see the work instead of getting an error

## References

original PR: https://github.com/otwcode/otwarchive/pull/3938

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*
agenderdanvers

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*
they/them
